### ISSUE TITLE
File-in-use issues with .NET Core 2.1 Preview 2

### ISSUE THREAD

**Issue by richlander** [dotnet/cli#8741]
I am running into file-in-use issues with .NET Core 2.1 Preview 2. The following is just one example. Didn't see this with preview 1.

C:\git\dotnet-docker\samples\dotnetapp\dotnetapp>dotnet run
C:\Program Files\dotnet\sdk\2.1.300-preview2-008308\NuGet.targets(114,5): error : Access to the path 'C:\git\dotnet-docker\samples\dotnetapp\utils\obj\project.assets.json' is denied. [C:\git\dotnet-docker\samples\dotnetapp\dotnetapp\dotnetapp.csproj]

The build failed. Please fix the build errors and run again.

**Comment by livarcocc** [comment]
Can you give us any more details?

Is this any special application? Just console app? Is this reproing 100% of the time for you? We use the preview2 CLI to build the CLI itself and haven't been running into this, so I suspect there is something special in your machine that we need to understand.

If this is still happening, I would love to get someone from my team to take a look at your machine.

**Comment by richlander** [comment]
Time didn't solve the problem. I waited about 20mins (while I did other things). I eventually rebooted my machine, which solved the issue. I tried to see if there was a running process that would have locked these files but could not find one.

I will see if it happens again. There is some chance that this was related to Docker because I had volume mounted the same directory on my machine into a container. I killed all the running containers and the problem persisted, so that didn't present itself as the obvious cause.

I'll close this now and re-open it if I see this again.
